### PR TITLE
Clean up repo

### PR DESCRIPTION
**Pull request by luan-dev**
## Changelog

- update dependencies for all packages
- update tests for all packages
- add github actions for ci/cd

### CODE REVIEW

**Review by rfricke-asymmetrik — DISMISSED** [review]
👍

**Review by ekdeveloper — DISMISSED** [review]

File: packages/fhir-sanitize-param/index.test.js [hunk 1]
```diff
@@ -118,12 +118,6 @@ describe('Standard Parameter Sanitization Tests', () => {
       };
       expect(callSanitizeNumber).toThrowError(/expected number for parameter foo/);
     });
-    test('Should reject invalid numbers', () => {
```
Comment (rfricke-asymmetrik) [inline packages/fhir-sanitize-param/index.test.js:121]: Was this test not needed anymore?

**Review by rfricke-asymmetrik — APPROVED** [review]

File: packages/fhir-sanitize-param/index.test.js (hunk 1 above)
Comment (luan-dev) [reply packages/fhir-sanitize-param/index.test.js:121]: the issue that this is testing was fixed by a dep update

File: packages/fhir-qb/index.js [hunk 2]
```diff
@@ -451,15 +451,6 @@ class QueryBuilder {
         tokenQuery = this.qb.buildAndQuery(queries);
         break;
       case 'Identifier':
```
Comment (dionboles-asym) [inline packages/fhir-qb/index.js:453]: Did you combined identifier and contact point into one case

**Review by dionboles-asym — APPROVED** [review]

File: packages/fhir-qb/index.js (hunk 2 above)
Comment (luan-dev) [reply packages/fhir-qb/index.js:453]: yes, it was just an exact copy

**Pull request merged**